response_operations_ui/controllers/message_controllers.py

```diff
@@ -37,3 +37,17 @@ def get_message_list(params):
         # user but not give a server error page.
         logger.exception("Response was successful but didn't contain messages element")
         return []
+
+
+def send_message(message_json):
+    logger.debug("Sending messsage")
+
+    url = f'{current_app.config["BACKSTAGE_API_URL"]}/v1/secure-message/send-message'
+    # This will be removed once UAA is completed.  For now we need the call to backstage to include
+    # an Authorization in its header a JWT that includes party_id and role.
+    encoded_jwt = jwt.encode({'user': 'BRES', 'party_id': 'BRES', 'role': 'internal'}, 'testsecret', algorithm='HS256')
+
+    response = requests.post(url, headers={'Authorization': encoded_jwt, 'Content-Type': 'application/json',
+                                           'Accept': 'application/json'}, data=message_json)
+    if response.status_code != 201:
```

Comment:
Following on from James' change, should this now be checking for !=200 rather than 201?